src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -50,7 +52,10 @@ public static function register($mode = 0)
             if (false === $mode) {
                 $mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
             }
-            if (DeprecationErrorHandler::MODE_WEAK !== $mode && (!isset($mode[0]) || '/' !== $mode[0])) {
+            if (!in_array($mode, array(
```

Comment:
I think it's a matter of taste, because to me the more readable version is of course mine. Plus it's less repetitive. I'll do as you say though, b/c you're probably more aware of what the sf style is.